drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/test/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/handlers/ScenarioSimulationBusinessCentralDocksHandlerTest.java

```diff
@@ -0,0 +1,202 @@
+        final UberfireDock testRunnerDock = (UberfireDock) docks.toArray()[MANAGED_DOCKS.REPORT.ordinal()];
+
+        scenarioSimulationBusinessCentralDocksHandlerSpy.expandTestResultsDock();
+        verify(authoringWorkbenchDocksMock).expandAuthoringDock(eq(testRunnerDock));
+    }
+
+    @Test
+    public void setScesimPath() {
+        final Collection<UberfireDock> docks = scenarioSimulationBusinessCentralDocksHandlerSpy.provideDocks("id");
+        final UberfireDock settingsDock = (UberfireDock) docks.toArray()[MANAGED_DOCKS.SETTINGS.ordinal()];
+        final UberfireDock toolsDock = (UberfireDock) docks.toArray()[MANAGED_DOCKS.TOOLS.ordinal()];
+        final UberfireDock cheatSheetDock = (UberfireDock) docks.toArray()[MANAGED_DOCKS.CHEATSHEET.ordinal()];
+        final UberfireDock coverageDock = (UberfireDock) docks.toArray()[MANAGED_DOCKS.COVERAGE.ordinal()];
+        String TEST_PATH = "TEST_PATH";
+        scenarioSimulationBusinessCentralDocksHandlerSpy.setScesimEditorId(TEST_PATH);
+        assertTrue(settingsDock.getPlaceRequest().getParameters().containsKey(SCESIMEDITOR_ID));
+        assertEquals(TEST_PATH, settingsDock.getPlaceRequest().getParameter(SCESIMEDITOR_ID, "null"));
+        assertTrue(toolsDock.getPlaceRequest().getParameters().containsKey(SCESIMEDITOR_ID));
+        assertEquals(TEST_PATH, toolsDock.getPlaceRequest().getParameter(SCESIMEDITOR_ID, "null"));
+        assertTrue(cheatSheetDock.getPlaceRequest().getParameters().containsKey(SCESIMEDITOR_ID));
+        assertEquals(TEST_PATH, cheatSheetDock.getPlaceRequest().getParameter(SCESIMEDITOR_ID, "null"));
+        assertTrue(coverageDock.getPlaceRequest().getParameters().containsKey(SCESIMEDITOR_ID));
+        assertEquals(TEST_PATH, coverageDock.getPlaceRequest().getParameter(SCESIMEDITOR_ID, "null"));
+    }
+
+    @Test
+    public void resetDocks() {
+        scenarioSimulationBusinessCentralDocksHandlerSpy.resetDocks();
+        verify(testRunnerReportingPanelWrapperMock, times(1)).reset();
+        verify(coverageReportPresenterMock, times(1)).reset();
+        verify(testToolsPresenterMock, times(1)).reset();
+        verify(settingsPresenterMock, times(1)).reset();
+        verify(cheatSheetPresenterMock, times(1)).reset();
+    }
+
+    @Test
+    public void getTestRunnerReportingPanel() {
+        IsWidget panel = scenarioSimulationBusinessCentralDocksHandlerSpy.getTestRunnerReportingPanelWidget();
+        assertSame(testRunnerReportingPanelWidgetMock, panel);
+    }
+
+    @Test
+    public void updateTestRunnerReportingPanelResult() {
+        TestResultMessage testResultMessageMock = mock(TestResultMessage.class);
+        scenarioSimulationBusinessCentralDocksHandlerSpy.updateTestRunnerReportingPanelResult(testResultMessageMock);
+        verify(testRunnerReportingPanelWrapperMock, times(1)).onTestRun(eq(testResultMessageMock));
+    }
+
+    @Test
+    public void getSettingsView() {
```

Comment:
Is that name is right? I see only coverage report View checks

Reply:
@dupliaka Done